Automattic/sensei
# Update color customization

Fixes #4589 

I did not add the parts from Blockbase that saves values from the Customizer into the user's FSE Global Styles settings to keep things simple. This means Customizer settings will override changes made in Global Styles. We plan to push users to use the Customizer for now as Global Styles is still a bit unrefined.

### Changes proposed in this Pull Request

* Make the customizer work with the override theme
* Add Background and Text color
* Update CSS to use customizer colors
* Replace hard-coded colors in CSS, fix a few things
* Use opacity for greys and borders, so they work with any background (dark or light)

### Testing instructions

* Enable the feature flag: `add_filter( 'sensei_feature_flag_course_theme', '__return_true' );`
* Open up a lesson in learning mode
* Click Customize, change colors. See them change.
* Publish changes and close customizer. Check that the colors are still there.


### Screenshot / Video

https://user-images.githubusercontent.com/176949/150037477-27c8207e-eb24-47db-9c8e-ac26a70bbd96.mov
